README.md

```diff
@@ -8,21 +8,55 @@ Welcome to the editing environment for the DE-TEL Book.
 
 ## Prepare Development Environment
 
-1. Install [Ruby](https://www.ruby-lang.org) on your machine ([Detailed instructions](https://jekyllrb.com/docs/installation/))
+__Step 1.__ Install [Ruby](https://www.ruby-lang.org) on your machine ([Detailed instructions](https://jekyllrb.com/docs/installation/))
 
-2. Clone this repository and open a terminal / command line
+If you are going to install Ruby on a Windows machine, there is a convenient tool [RubyInstaller](https://rubyinstaller.org/downloads/).
 
-   Check that Ruby is successfully installed and can be accessed from the terminal / command line.
-   To do this, run the command "ruby -v".
-   It should return the installed version of Ruby.
+Ruby version 2.7.5 is recommended to work with this repository.
 
-3. In the terminal/command line, navigate to your main project folder
+__Step 2.__ Check that Ruby is successfully installed and can be accessed from the terminal / command line.
+To do this, open the Command prompt and run command:
+   
+    ruby -v
+   
+It should return the installed version of Ruby.

```

Comment:
On Windows, there can be the case that Ruby is successfully installed but the command prompt does not have access to it.
The advanced option would be to add the installation path of Ruby to Windows' PATH variable but the simpler solution is to just use a specially prepared command prompt that is installed with Ruby and which can be found in the start menu.

```suggestion
It should return the installed version of Ruby.

On Windows, you can find an entry called "Start Command Prompt with Ruby" in your start menu after the installation. It makes sure that Ruby is accessible by that command prompt. If the default command prompt cannot find Ruby, you can keep using this special command prompt.
```